src/Components/CreateNewOption.tsx

```diff
@@ -1,49 +1,98 @@
-import { useState } from 'react'
 import { CategoryInput } from './CategoryInput'
 import styled from '@emotion/styled'
+import { css } from '@emotion/react'
 import { FillBtn } from './basic/button/Button'
-import { TextInput } from './basic/InputBox'
-import { TagBtn } from './basic/button/TagButton'
-import { Label } from './basic/Label'
 
-export const CreateNewOption = () => {
-  const [isAns, setIsAns] = useState<boolean>(true)
+interface propsType{
+  isAnswer: boolean
+  setIsAnswer: (item:boolean)=>void
+  setOption: (item:string)=>void
+  setKeywords: (item:string[])=>void
+  onSubmit: ()=>void
+}
+
+export const CreateNewOption = (props:propsType) => {
 
-  function getCategory(cats: string[]) {
-    console.log(cats)
+  function updateSuggested(e:React.ChangeEvent<HTMLInputElement>) {
+    props.setOption(e.target.value)
   }
 
   return (
     <div>
-      <Container>
-        <div>
-          <Label text="Create New Option" color="blue" size={0} />
-          <TagBtn onClick={() => setIsAns(true)} id={isAns ? 'AnsAct' : 'Ans'}>
-            Answer
-          </TagBtn>
-          <TagBtn onClick={() => setIsAns(false)} id={!isAns ? 'DistAct' : 'Dist'}>
-            Distractor
-          </TagBtn>
-        </div>
-        <Block>
-          <TextInput placeholder="Suggest an answer or distractor for this question" />
-          <CategoryInput getCategory={getCategory} />
-        </Block>
-        <FillBtn>Submit</FillBtn>
-      </Container>
+      <h3>Create New Option</h3>
+      <Toggles>
+        <ToggleBtn onClick={() => props.setIsAnswer(true)} id={props.isAnswer ? 'AnsAct' : 'Ans'}>
+          <strong>Answer</strong>
+        </ToggleBtn>
+        <ToggleBtn onClick={() => props.setIsAnswer(false)} id={!props.isAnswer ? 'DistAct' : 'Dist'}>
+          <strong>Distractor</strong>
+        </ToggleBtn>
+      </Toggles>
+      <div>
+        <Input type="text" placeholder="Suggest an answer or distractor for this question" onChange={updateSuggested}/>
+        <CategoryInput getCategory={props.setKeywords} />
+        <FillBtn onClick={props.onSubmit}>Submit</FillBtn>
+      </div>
     </div>
   )
 }
 
-const Container = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: 20px;
-  padding: 10px 0 10px 0;
+const Input = styled.input`
+  padding: 16px;
+  border-radius: 6px;
+  border: 1px solid #bdbdbd;
+  width: 100%;
+  box-sizing: border-box;
+  font-size: 16px;
+  &:placeholder {
+    color: #b7bfc7;
+  }
+  &:focus {
+    outline: none;
+    border-color: #212121;
+  }
 `
 
-const Block = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: 12px;
+const Toggles = styled.div`
+  margin-bottom: 20px;
 `
+
+const ToggleBtn = styled.div<{ id: 'Ans' | 'AnsAct' | 'Dist' | 'DistAct' | 'SubmitBtn' }>`
```

Comment:
better write as 
```suggestion
const ToggleBtn = styled.div<{ id: 'Ans' | 'AnsAct' | 'Dist' | 'DistAct' | 'SubmitBtn' }>`${({ id }) => css`
border-color: ${id === 'Ans' || id === 'AnsAct' ? 'rgb(30, 144, 30)' : 'rgb(220, 51, 51)'};
`}
```

Reply:
> better write as

Updated the code to follow the new design; "ToggleBtn" is now removed:)